src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -104,7 +104,6 @@
 
   <!-- Sources -->
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\Internal\Console.cs" />
```

Comment:
Delete  ` <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetStdHandle.cs">` and friends in this file, and them to the Windows-specific section of the shared CoreLib project instead.